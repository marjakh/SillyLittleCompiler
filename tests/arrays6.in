let a = new Array(3);
let i = 0;
let b = 10;
while (i < 3) {
   a[i] = 600 + b;
  i = i + 1;
}
i = 0;
while (i < 3) {
  write(a[i]);
  i = i + 1;
}
