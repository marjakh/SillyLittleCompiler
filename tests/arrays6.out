610
610
610
